packages/coordinator/src/services/translation/request/prescription.ts

```diff
@@ -317,10 +317,12 @@ export function parseNumberOfRepeatsAllowed(numberOfRepeatsAllowed: string | Los
     ? parseInt(numberOfRepeatsAllowed)
     : numberOfRepeatsAllowed.valueOf()
   if (typeof numberOfRepeatsAllowedNumber === "bigint") {
-    const numberOfRepeatsAllowedString = numberOfRepeatsAllowedNumber.toString()
+    const numberOfRepeatsAllowedString = numberOfRepeatsAllowedNumber as unknown as string
```

Comment:
I do not like this line, if anyone knows a nicer fix? Seemed to be issues between converting `bigints` to `strings` and this is where a goose chase lead me, same for line 324